src/randgraphs.jl

```diff
@@ -96,11 +96,10 @@ end
 
 function _suitable(edges::Set{Edge}, potential_edges::Dict{Int, Int})
     isempty(potential_edges) && return true
-    for (s1, s2) in combinations(collect(keys(potential_edges)), 2)
-        if (s1 > s2)
-            s1, s2 = s2, s1
-        end
-        ∉(Edge(s1, s2), edges) && return true
+    list = keys(potential_edges)
+    for s1 in list, s2 in list
+        s1 >= s2 && continue
+        (Edge(s1, s2) ∉ edges) && return true
```

Comment:
Does this change take it from n^2 to n choose 2 times around the loop? That way we don't have to pay the loop overhead for all the continues.
```julia
for i,s1 in enumerate(list)
   for j in i:length(list)
       s2=list[j]
      ... rest of the code
end
end
```